recipes/mmult/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "MMULT" %}
+{% set version = "0.0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/lijinbio/MMULT/archive/v{{ version }}.tar.gz
+  sha256: d39aeeccd3f0aa5851a9db38369ae7beb8ffc0389f65340fc786a4dcff2f49b7
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - boost
```

Comment:
boost, eigen and co do not need to be in the build section

Reply:
@bgruening Thanks. Those runtime dependencies are removed from the build section.